src/modules/searchResults/ui.js

```diff
@@ -8,9 +8,13 @@ export default function(product,idx,swatchUI,productViewType,products ){
         unxDescription
     } = product; 
     const {
-        productItemClass
+        productItemClass,
+        defaultImage
     }  = products;
-    const imgUrl = Array.isArray(unxImageUrl) ? unxImageUrl[0]:unxImageUrl;
+    let imgUrl = Array.isArray(unxImageUrl) ? unxImageUrl[0]:unxImageUrl;
+    if(!imgUrl) {
+        imgUrl = defaultImage;
+    }
```

Comment:
Update the package.json version & mention the change in the changelog. Also add the new option in the readme file table